scripts/goldpolish-target-extract-seq.py

```diff
@@ -109,27 +119,28 @@ def extract_subsequences_from_bed(sequence, name, length, writer, coords):
     if name in coords:
         count = 0
         coord_list = coords[name]
-        idx = 0
-
-        while idx < len(coord_list) - 1:
-            # distance between coords of seqs to be extracted
-            inter_seq_len = int(coord_list[idx + 1][0]) - int(coord_list[idx][1])
-            if inter_seq_len < 2 * length:
-                start = coord_list[idx][0]
-                end = coord_list[idx + 1][1]
-
-                list_before_index = coord_list[:idx] if idx > 0 else []
-                list_after_index = (
-                    coord_list[idx + 2 :] if idx < len(coord_list) + 2 else []
-                )
-                coord_list = (
-                    list_before_index + [None] + [(start, end)] + list_after_index
-                )
+        idx = 1
+
+        filtered_coords = (
+            []
+        )  # short uppercase seqs are appended to adjacent coordinates
+        filtered_coords.append(
+            coord_list[0]
+        )  # first item always appended, avoids 0-index issues
+
+        while idx < len(coord_list):
+            coord = Coordinate(coord_list[idx])
```

Comment:
```suggestion
            coord = Coordinate(*coord_list[idx])
```
By adding this dereference, you can change your class/namedtuple to have two arguments (start and end) instead, which is more readable